packages/ethereum-contracts/contracts/mocks/SuperfluidDesctructorMock.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: AGPLv3
+// solhint-disable
+pragma solidity 0.7.6;
+
+contract SuperfluidDesctructorMock {
```

Comment:
not a huge deal but a small typo: `SuperfluidDesctructorMock` => `SuperfluidDestructorMock` same w/ file name

Reply:
fixed